.circleci/config.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       - install_js
       - run:
           name: Eslint
-          command: yarn eslint:ci
+          command: yarn lint:ci
```

Comment:
It might be better to fix this in the core repo. I mean eslint seems clearer 😁

Reply:
But we have different linters: `markdownlint`, `jsonlint`, `stylelint` (this could be [removed](https://github.com/mui-org/material-ui-x/pull/3259#issuecomment-975509724) I think).

Reply:
Done